docs/admin/index.md

```diff
@@ -0,0 +1,146 @@
+
+Previous `docker-compose` assumes you already have Orion running somewhere else and you just want to deploy Perseo.
+However, you can easily add the definition of the `orion:` and `orion-mongo:` services in the `docker-compose.yml` file.
+You will also need to change the references to them in the `environment:` section of the `perseo-fe` service.
+
+### Service Environment Variables
+
+Perseo service can be configured using the following environment variables. Please note variables marked with ✅ are
+required.
+
+#### perseo-fe
+
+| Environment variable      | Default Value | Description                                                         |
+| :------------------------ | :------------ | :------------------------------------------------------------------ |
+| `PERSEO_ENDPOINT_HOST`    |               | Host where the CEP will listen to.                                  |
+| `PERSEO_ENDPOINT_PORT`    |               | Port where the CEP will listen to.                                  |
+| `PERSEO_MONGO_ENDPOINT`✅ |               | Endpoint (host[:port]) list for Mongo DB.                           |
+| `PERSEO_MONGO_REPLICASET` |               | ReplicaSet name for Mongo DB.                                       |
+| `PERSEO_MONGO_USER`       |               | User for Mongo DB.                                                  |
+| `PERSEO_MONGO_PASSWORD`   |               | Password for Mongo DB.                                              |
+| `PERSEO_CORE_URL`✅       |               | URL where Perseo Core is listening (e.g: http://host_or_ip:port).   |
+| `PERSEO_NEXT_URL`         |               | URL where Perseo Core replicated node is listening.                 |
+| `PERSEO_ORION_URL`✅      |               | URL of the Orion Context Broker.                                    |
+| `PERSEO_LOG_LEVEL`        |               | Log level.                                                          |
+| `PERSEO_SMTP_HOST`        |               | Host of the SMTP server                                             |
+| `PERSEO_SMTP_PORT`        |               | Port of the SMTP server                                             |
+| `PERSEO_SMTP_VERIFY_CA`   | `false`       | `true` if self-signed or invalid TLS certificate should be rejected |
+| `PERSEO_SMTP_SECURE`      | `false`       | `true` if SSL should be used with the SMTP server                   |
+| `PERSEO_SMTP_AUTH_USER`   |               | Authentication data, the username                                   |
+| `PERSEO_SMTP_AUTH_PASS`   |               | Authentication data, the user password                              |
+| `PERSEO_NOTICES_PATH`     | `'/notices'`  | Path for incoming notices                                           |
+| `PERSEO_RULES_PATH`       | `'/rules'`    | Path for incoming rules                                             |
+
+-   For legacy SMS and SMPP support, please refer to [this file](configuration.md).
+
+#### perseo-core
```

Comment:
Why this env var is in separate secction?

Reply:
The first section shows only perseo-fe env vars, and the second one is for perseo-core. After checking the variables supported by perseo-core, I have added MAX_AGE to the second table, a previous explanatory text and a reference to docker-compose.yml as example. Fixed in fe941d4ef3318f58d26480bfdcf475ed07e4773b